### PR TITLE
Adding clusterroles feature. Fixes #51

### LINKED ISSUE #51: Add support for Cluster roles
Add support for Cluster roles

### PR DESCRIPTION
**Pull request by shinusuresh**

### CODE REVIEW

File: src/ui/resource_tabs.rs [hunk 1]
```diff
@@ -34,6 +34,7 @@ static SECRETS_TITLE: &str = "Secrets";
 static RPL_CTRL_TITLE: &str = "ReplicationControllers";
 static STORAGE_CLASSES_LABEL: &str = "StorageClasses";
 static ROLES_TITLE: &str = "Roles";
+static CLUSTER_ROLES_TITLE: &str = "Cluster Roles";
```
Comment (deepu105) [inline src/ui/resource_tabs.rs:37]: Remove space 
```suggestion
static CLUSTER_ROLES_TITLE: &str = "ClusterRoles";
```

File: src/ui/resource_tabs.rs [hunk 2]
```diff
@@ -1057,6 +1059,54 @@ fn draw_roles_block<B: Backend>(f: &mut Frame<'_, B>, app: &mut App, area: Rect)
...
+  draw_resource_block(
+    f,
+    area,
+    ResourceTableProps {
+      title,
+      inline_help: DESCRIBE_YAML_AND_ESC_HINT.into(),
+      resource: &mut app.data.clusterroles,
+      table_headers: vec!["Name", "Age"],
+      column_widths: vec![Constraint::Percentage(40), Constraint::Percentage(20)],
+    },
```
Comment (deepu105) [inline src/ui/resource_tabs.rs:1097]: Colums should add up to 100

**Review by deepu105 — CHANGES_REQUESTED** [review]

File: src/ui/resource_tabs.rs (hunk 2 above)
Comment (shinusuresh) [reply src/ui/resource_tabs.rs:1098]: Done

File: src/ui/resource_tabs.rs (hunk 1 above)
Comment (shinusuresh) [reply src/ui/resource_tabs.rs]: Done

**Review by deepu105 — APPROVED** [review]

**Pull request merged**